notebooks/hf_llama_generation_example.ipynb

```diff
@@ -222,7 +252,7 @@
      "output_type": "stream",
      "text": [
       "[{'generated_text': 'I believe the meaning of life is to find happiness and fulfillment. Here are some of the things that bring me joy and fulfillment:\\n\\n'}]\n",
-      "24.2 s ± 0 ns per loop (mean ± std. dev. of 1 run, 1 loop each)\n"
+      "21.5 s ± 0 ns per loop (mean ± std. dev. of 1 run, 1 loop each)\n"
```

Comment:
the numbers for eager and  compile are practically the same, any reason for that?

Reply:
This notebook will be superseded by a new one, will check once updated.